src/components/EditAbilityScores.js

```diff
@@ -0,0 +1,135 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+
+import 'airbnb-js-shims' // to allow jest to understand Object.entries for parsing the character objects
+
+import { Form, FormGroup, Col, Button, ButtonGroup, ControlLabel } from 'react-bootstrap'
+
+export default class EditAbilityScores extends Component {
+  constructor (props) {
+    super(props)
+
+    this.renderAbilityScorePackages = this.renderAbilityScorePackages.bind(this)
+    this.pointsRemaining = this.pointsRemaining.bind(this)
+    this.increaseScore = this.increaseScore.bind(this)
+    this.decreaseScore = this.decreaseScore.bind(this)
+    this.resetScores = this.resetScores.bind(this)
+  }
+
+  pointsRemaining () {
+    let points = 27
+
+    Object.values(this.props.abilityScores).map((score) => {
```

Comment:
A properly advanced JS dev knows all the functional programming methods that were added to Array, including `forEach`, `map`, `reduce`, `reduceRight`, `filter`, `find`, and `some`